### PR TITLE
Create empty Saselburg file

### PR DESCRIPTION
**Pull request by ghost**
Adding new city file

### CODE REVIEW

**Comment by sjmp2** [comment]
@FailbetterBarry  https://en.wikipedia.org/wiki/Sasolburg ?

**Comment by ghost** [comment]
@SamFailbetter yes ><

To be fair I was only born there!

File: Barry-Saselburg.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#Saselburg
+
+### Things to do
+Oil refinery
+
+### Places to eat
+Don't know
```
Comment (sjmp2) [inline Barry-Saselburg.md]: Sounds awful, mate

Comment (sjmp2) [reply Barry-Saselburg.md]: @FailbetterBarry - The Google Images pics are terrifying

**Comment by sjmp2** [comment]
@FailbetterBarry - Where can I get Billtong?

**Pull request merged**